pkg/controllers/dynakube/phase.go

```diff
@@ -12,6 +12,22 @@ import (
 )
 
 func (controller *Controller) determineDynaKubePhase(dk *dynakube.DynaKube) status.DeploymentPhase {
+	components := []func(dk *dynakube.DynaKube) status.DeploymentPhase{
```

Comment:
@aorcholski could you please extend unit-tests to add similar tests to TestActiveGatePhaseChanges but for EEC and OteLC.

Reply:
done

Reply:
well done Adam! I like it